src/utils.ts

```diff
@@ -241,14 +219,10 @@ export const getTimeStamp = function() {
  * Retrieves list of swagger files to be processed for linting
  * @returns {Array} list of files to be processed for linting
  */
-export const getConfigFilesChangedInPR = function() {
-  if (prOnly === 'true') {
-    let targetBranch, cmd, filesChanged;
+export const getConfigFilesChangedInPR = async (pr: avocado.PullRequestProperties | undefined) => {
+  if (pr !== undefined) {
     try {
-      targetBranch = getTargetBranch();
-      execSync(`git fetch origin ${targetBranch}`);
-      cmd = `git diff --name-only HEAD $(git merge-base HEAD FETCH_HEAD)`;
-      filesChanged = execSync(cmd, { encoding: 'utf8' }).split('\n');
+      let filesChanged = (await pr.diff()).map(file => file.path)
```

Comment:
please add comments in avocado PullRequestUtils on what diff does and what it outputs.